lib/context.js

```diff
@@ -18,6 +18,7 @@ function Context (schema, handler, Reply, Request, contentTypeParser, config, er
   this.onResponse = null
   this.config = config
   this.errorHandler = errorHandler || defaultErrorHandler
+  this.isDefaultErrorHandler = !errorHandler
```

Comment:
This is not needed anymore, right?

Reply:
That's correct my mind just skipped over that.